src/System.Globalization/tests/DateTimeFormatInfo/DateTimeFormatInfoCalendarWeekRule.cs

```diff
@@ -11,64 +12,35 @@ namespace System.Globalization.Tests
 {
     public class DateTimeFormatInfoCalendarWeekRule
     {
-        // TestGetter: Call CalendarWeekRule getter method should return correct value for InvariantInfo
-        [Fact]
-        public void TestGetter()
-        {
-            VerificationHelper(DateTimeFormatInfo.InvariantInfo,
-                    CalendarWeekRule.FirstDay,
-                    false);
-        }
-
-        // TestSetter: Call CalendarWeekRule setter method should return correct value
-        [Fact]
-        public void TestSetter()
+        public static IEnumerable<object[]> CalendarWeekRule_TestData()
```

Comment:
You could consider just moving to [InlineData] and taking a String as the first argument and doing `new CultureInfo(<arg>).DateTimeFormat`.  In this case, the string for invariant is `""` (the empty string).  Might reduce some of the ceremony here.